src/pages/home/ReportScreen.js

```diff
@@ -517,10 +569,12 @@ function ReportScreen({
                                 style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
                                 onLayout={onListLayout}
                             >
-                                {isReportReadyForDisplay && !isLoadingInitialReportActions && !isLoading && (
+                                {isReportReadyForDisplay && (
```

Comment:
It's not clear to me why you were able to remove these extra conditions for rendering `ReportActionsView`